python/requirements.txt

```diff
@@ -52,7 +52,7 @@ scikit-image>=0.21.0
 prometheus_client>=0.7.1
 pandas
 tensorboardX
-aiohttp>=3.7
+aiohttp==3.11.12
```

Comment:
why changing this, this should be in sync with `setup.py`

Reply:
If I don't pin the version of aiohttp, it'll resolve to 3.11.14, which somehow breaks the existing tests.

Reply:
Okay let me create another PR and do some experiments on it. Converted this PR back to draft for now.

Reply:
I reran the tests again with aiohttp 3.11.14 and they passed. I think it might be due to some flaky tests or environment changes in recent commits. I've marked this PR as ready for review again.